apps/web/test/web_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule WebTest do
```

Comment:
Should probably have actual tests instead of a placeholder

Reply:
I'm not sure how I would test this. There is only one function in strip_server_header that has a non-default implementation and that is the info method. It does a Map.drop on the header field. I could create an implementation class that does a Map.drop() and then test that and have info call that function.

Reply:
I've written integration test for andi, discovery_api, and discovery_streams to test this functionality.